.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: GitHub CI
+
+on: [push]
+
+# Use bash explicitly for being able to enter the conda environment
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build-and-test:
+    name: Build Env, Install, Unit Tests
+    runs-on: ubuntu-latest
+    permissions:
+      # For publishing results
+      checks: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10"]
```

Comment:
Is this something we then have to update manually for each release?

Reply:
For each *Python* release, yes.